### PR TITLE
VEOSS-850| using the new endpoint returning the returns view model

### PR DESCRIPTION
**Pull request by oliwiaKot**

### CODE REVIEW

File: app/viewmodels/yourAccount/Returns.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.yourAccount
+
+import play.api.libs.json.{Json, OFormat}
+
+
+case class Returns(
```
Comment (DevAndyroo) [inline app/viewmodels/yourAccount/Returns.scala:22]: Can we name this a bit more descriptive? Maybe something like OpenReturns

File: app/views/IndexView.scala.html [hunk 2]
```diff
@@ -69,70 +68,70 @@ <h2 class="govuk-heading-m">@messages("index.yourReturns")</h2>
...
+                <p class="govuk-body">@messages("index.yourReturns.returnOverdue", returnsViewModel.overdueReturns.head.period.displayText)</p>
+                <p class="govuk-body">@messages("yourAccount.return.inProgress", inProgress.period.displayText)</p>
                 }
                 case (Some(inProgress), None, x) => {
                 <p class="govuk-body">@messages("index.yourReturns.onlyReturnsOverdue", x)</p>
-                <p class="govuk-body">@messages("yourAccount.return.inProgress", inProgress.displayText)</p>
+                <p class="govuk-body">@messages("yourAccount.return.inProgress", inProgress.period.displayText)</p>
                 }
 
                 case (None, Some(period), 0) => {
```
Comment (DevAndyroo) [inline app/views/IndexView.scala.html:84]: Can we rename this variable to be return instead of period (and as with the other ones in this area)

**Review by DevAndyroo — CHANGES_REQUESTED** [review]

File: app/views/IndexView.scala.html (hunk 2 above)
Comment (oliwiaKot) [reply app/views/IndexView.scala.html]: done

File: app/viewmodels/yourAccount/Returns.scala (hunk 1 above)
Comment (oliwiaKot) [reply app/viewmodels/yourAccount/Returns.scala]: done

**Review by DevAndyroo — APPROVED** [review]

**Pull request merged**